Cargo.toml

```diff
@@ -21,61 +21,61 @@ exclude = [
 
 [workspace.package]
 edition = "2021"
-version = "0.14.0"
+version = "0.15.0"
```

Comment:
I wouldn't increase the version of the workspace, burn `0.14.0` isn't released yet, so it's already the next burn version.

Reply:
Reverted.